syntax = "proto3";

package chatroom.msgpb;

message UserLogin {
    uint32 account = 1;
    string password = 2;
}